mediaengine.go

```diff
@@ -233,6 +250,23 @@ func NewRTPH264Codec(payloadType uint8, clockrate uint32) *RTPCodec {
 	return c
 }
 
+// NewRTPH264CodecExt is a helper to create an H264 codec
+func NewRTPH264CodecExt(payloadType uint8, clockrate uint32) *RTPCodec {
+	c := NewRTPCodecExt(RTPCodecTypeVideo,
+		H264,
+		clockrate,
+		0,
+		"level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42001f",
```

Comment:
Can you make this a `const` we shouldn't hardcode this, but I can fix that later!

Reply:
Maybe you fix that better

Reply:
agree! I will break this in `/v3` I need to revisit how things are done with setting outbound tracks.